dockerfiles/opensuse/opensuse-42.3/opensuse-42.3-builder/README

```diff
@@ -0,0 +1,2 @@
+The Dockerfile for this directory is autogenerated based on dockerfiles/templates/Dockerfile.builder
```

Comment:
The builder directories were completely removed with a previous commit. So there isn't a need to add this anymore.